docs/spec/protocol.rst

```diff
@@ -181,10 +178,10 @@ subtyping -- the semantics of inheritance is not changed. Examples::
     represent(nice) # OK
     represent(another) # Also OK
 
-Note that there is little difference between explicit and implicit
+Note that there is little difference between explicit and implicit consistent
```

Comment:
Again not sure adding "consistent" here makes sense. Perhaps instead we can talk about "subclasses" of the protocol.

Reply:
I hadn't meant to leave this open, just got called away suddenly earlier before I'd gotten through all the feedback. I changed this to use similar wording as elsewhere.